### DISCUSSION TITLE
What is wrong with my CSS?

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#60450] [Programming Help] [answered]
### Body

https://github.com/linknprk/linknprk.github.io/blob/main/index.css
https://github.com/linknprk/linknprk.github.io/blob/main/chester.html

it’s open source so you can just edit the code

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by emullallphotography** [comment]
The CSS was supposed to align the .main in chester.html but it didn’t

**Reply by GhostOf0days** [reply, marked as answer]
There is no .main. The . syntax is used for targeting classes. The # syntax is used for targeting ID's. You do not have a class named main, so you would have to add it. For similar reasons, in the CSS, you need to rename .sites to .links or change the class name to 'sites' for the 'links' class in your HTML.

**Comment by jshtuazon2** [comment]
✓
